examples/text_summarization/extractive_summarization_cnndm_transformer.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>1) BERTSum paper hyperlink needs to be fixed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>